http-client-core/src/main/java/io/micronaut/http/client/HttpClientConfiguration.java

```diff
@@ -237,6 +237,8 @@ public void setHttpVersion(HttpVersion httpVersion) {
     }
 
     /**
+     * [not in java.net HttpClient].
```

Comment:
why is this only for netty?

Reply:
This is for setting the internal Netty log level.

There is nothing I can find in the java HttpClient, other than setting a system property to turn on logging

ie:

https://github.com/micronaut-projects/micronaut-core/blob/a4abbecd37c982157b2335a43f0dde13e0317dc2/http-client-javanet/build.gradle.kts#L22

A lot of HttpClient seems to be hidden away, and only programmable via system properties :-(